src/components/donation-targets/index.tsx

```diff
@@ -20,12 +29,11 @@ interface Props {
 }
 const DonationTargets: FunctionComponent<Props> = props => (
     <div className={styles.stats}>
-        <h3>{props.charity.longName} Donation Targets</h3>
+        <h3>{props.charity.longName} <span className='light'>Donation Targets</span></h3>
```

Comment:
I'm assuming you can put the classname in quotes because youve declared it as a global, just want to confirm that is the reason

Reply:
That is the reason